.github/workflows/preDeploy.yml

```diff
@@ -211,49 +224,43 @@ jobs:
   # Check if actor is member of Expensify organization by looking for Expensify/expensify team
   isExpensifyEmployee:
     runs-on: ubuntu-latest
+    needs: getMergedPullRequest
 
     outputs:
-      IS_EXPENSIFY_EMPLOYEE: ${{ fromJSON(steps.checkActor.outputs.isTeamMember) }}
```

Comment:
I just wanted to confirm here, when the output is from a step in the job, we do not have to use the `fromJSON`, right? It has been like that in other place.

The `fromJSON` is used if it is output across jobs?